### PR TITLE
Attempt to Fix Visa Cal scraper

### PR DESCRIPTION
**Pull request by ifeins**
I've applied the changes suggested by @sagiba in https://github.com/eshaham/israeli-bank-scrapers/issues/200 but I'm still unable to get the scraper to work.

It fails with the following output:

```
➜  israeli-bank-scrapers git:(fix_visa_cal_scraper) ✗ npm start

> israeli-bank-scrapers@0.6.7 start /Users/ifeins/Development/israeli-bank-scrapers
> node -r babel-register playground/scrape.js

Visa Cal: START_SCRAPING
Visa Cal: INITIALIZING
Visa Cal: LOGGING_IN
Auth result: "שם המשתמש או הסיסמה שהוזנו שגויים"
Visa Cal: TERMINATING
Visa Cal: END_SCRAPING
success: false
error type: generic
error: body used already for: https://connect.cal-online.co.il/api/authentication/login
```

@sagiba Can you let me know if I've missed anything?

### CODE REVIEW

File: src/scrapers/visa-cal.js [hunk 1]
```diff
@@ -145,19 +147,19 @@ function prepareTransactions(txns, startMoment, combineInstallments) {
 
 async function getBankDebits(authHeader, accountId) {
   const bankDebitsUrl = getBankDebitsUrl(accountId);
-  return fetchGet(bankDebitsUrl, authHeader);
+  return fetchGet(bankDebitsUrl, Object.assign(authHeader, HEADER_SITE));
```
Comment (esakal) [inline src/scrapers/visa-cal.js]: Make sure you update the original authHeader as following:
1. revert this change so you will pass `authHeader` as is.
2. change line in method `fetchData` line 274 from 
```
    const authHeader = { Authorization: this.authHeader };	
```
 to 
```
    const authHeader = Object.assign({ Authorization: this.authHeader }, HEADER_SITE);
```



**NOTE**: This comment is relevant to all places where you changed `authHeader` into `Object.assign(authHeader, HEADER_SITE)`.

**Comment by sagiba** [comment]
@esakal, I recommend you edit your comments above and remove the last 4 digits of your credit card, you don't want to have those publicly linked to your name as they are often used for security verification purposes.

**Comment by esakal** [comment]
@sagiba thanks I removed that image.

**Comment by esakal** [comment]
closing this pr - will be completed in #202
